server/api/rounds.py

```diff
@@ -256,13 +256,13 @@ def cvrs_for_contest(contest: Contest) -> supersimple.CVRS:
 
 def sampled_ballot_interpretations_to_cvrs(contest: Contest) -> supersimple.CVRS:
     def interpretation_to_cvr_value(interpretation: Interpretation) -> int:
-        if interpretation == Interpretation.BLANK:
-            return 0
-        elif interpretation == Interpretation.VOTE:
+        if interpretation == Interpretation.VOTE:
             return 1
+        elif interpretation == Interpretation.BLANK:
+            return 0  # TODO make this a vote for the loser?
         else:
             assert interpretation == Interpretation.CANT_AGREE
-            return 0  # TODO what to do here?
+            return 0  # TODO make this a vote for the loser?
```

Comment:
This is a more interesting case. @mcchilders how has this been handled in the past?